server/sonar-web/src/main/js/apps/about/components/AboutAppForSonarQubeDotComLazyLoader.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import React from 'react';
+
+export default class AboutAppForSonarQubeDotComLazyLoader extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      AboutAppForSonarQubeDotCom: null
+    };
+  }
+
+  componentDidMount() {
+    if (!this.state.AboutAppForSonarQubeDotCom) {
+      require.ensure(['./AboutAppForSonarQubeDotCom'], require => {
```

Comment:
This is a bit weird api, but you don't have to put anything for the first parameter (`[]` is fine). Webpack will parse the code inside the `require.ensure`, find all calls to `require` and make sure dependencies are available.